src/ucs/arch/cpu.c

```diff
@@ -72,13 +77,23 @@ const ucs_cpu_builtin_memcpy_t ucs_cpu_builtin_memcpy[UCS_CPU_VENDOR_LAST] = {
 };
 
 const size_t ucs_cpu_est_bcopy_bw[UCS_CPU_VENDOR_LAST] = {
-    [UCS_CPU_VENDOR_UNKNOWN]     = 5800 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_INTEL]       = 5800 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_AMD]         = 5008 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_GENERIC_ARM] = 5800 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_GENERIC_PPC] = 5800 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_FUJITSU_ARM] = 12000 * UCS_MBYTE,
-    [UCS_CPU_VENDOR_ZHAOXIN]     = 5800 * UCS_MBYTE
+    [UCS_CPU_VENDOR_UNKNOWN]     = UCS_CPU_EST_BCOPY_BW_DEFAULT,
+    [UCS_CPU_VENDOR_INTEL]       = UCS_CPU_EST_BCOPY_BW_DEFAULT,
+    [UCS_CPU_VENDOR_AMD]         = UCS_CPU_EST_BCOPY_BW_AMD,
+    [UCS_CPU_VENDOR_GENERIC_ARM] = UCS_CPU_EST_BCOPY_BW_DEFAULT,
+    [UCS_CPU_VENDOR_GENERIC_PPC] = UCS_CPU_EST_BCOPY_BW_DEFAULT,
+    [UCS_CPU_VENDOR_FUJITSU_ARM] = UCS_CPU_EST_BCOPY_BW_FUJITSU_ARM,
+    [UCS_CPU_VENDOR_ZHAOXIN]     = UCS_CPU_EST_BCOPY_BW_DEFAULT
+};
+
+const size_t ucs_cpu_est_bcopy_bw_proto[UCS_CPU_VENDOR_LAST] = {
+    [UCS_CPU_VENDOR_UNKNOWN]     = UCS_CPU_EST_BCOPY_BW_PROTO_DEFAULT,
+    [UCS_CPU_VENDOR_INTEL]       = UCS_CPU_EST_BCOPY_BW_PROTO_DEFAULT,
+    [UCS_CPU_VENDOR_AMD]         = UCS_CPU_EST_BCOPY_BW_AMD,
+    [UCS_CPU_VENDOR_GENERIC_ARM] = UCS_CPU_EST_BCOPY_BW_PROTO_DEFAULT,
+    [UCS_CPU_VENDOR_GENERIC_PPC] = UCS_CPU_EST_BCOPY_BW_PROTO_DEFAULT,
+    [UCS_CPU_VENDOR_FUJITSU_ARM] = UCS_CPU_EST_BCOPY_BW_FUJITSU_ARM,
+    [UCS_CPU_VENDOR_ZHAOXIN]     = UCS_CPU_EST_BCOPY_BW_PROTO_DEFAULT
```

Comment:
it's not so nice that low-level UCS layer is aware of protocol version that is in upper layer - UCP
maybe we could change UCS to have the protov2 estimations, and also add protov1 table in UCP, that will be used when protocols v2 are disabled?